common/src/main/java/org/dash/wallet/common/data/entity/ExchangeRate.kt

```diff
@@ -79,13 +88,16 @@ open class ExchangeRate : Parcelable {
             // If the currency is not a valid ISO 4217 code, then set the
             // currency name to be equal to the currency code
             // exchanges often have "invalid" currency codes like USDT and CNH
-            currencyName = if (currencyCode.length == 3) {
-                try {
-                    getCurrency().displayName
-                } catch (x: IllegalArgumentException) {
+            currencyName =
+                if (currencyCode.length == 3) {
+                    try {
+                        getCurrency().displayName
+                    } catch (x: IllegalArgumentException) {
+                        currencyCode
+                    }
+                } else {
                     currencyCode
                 }
```

Comment:
Here, I am sure that the ktlint plugin on automatically formatted this file upon save.  I have turned off this feature.